packages/ng/filter-pills/filter-pills-bar/filter-pills-bar.component.html

```diff
@@ -0,0 +1,53 @@
+<lu-scroll-box class="filterPillBar-scrollBox">
+	<div class="filterPillBar-scrollBox-first"></div>
+
+	<div class="filterPillBar-scrollBox-group">
+		@if (addonBefore()) {
+		<ng-container *ngTemplateOutlet="addonBefore()"></ng-container>
+
+		<lu-divider class="filterPillBar-divider" />
+		} @if (optionalPills().length > 0) {
+		<button
+			class="filterPill mod-button"
+			type="button"
+			[luTooltip]="intl.additionalFilters"
+			luTooltipOnlyForDisplay
+			[luPopover2]="contentOptionsTpl"
+			luPopoverNoCloseButton
+			[customPositions]="popoverPositions"
+		>
+			<lu-icon class="filterPill-icon" icon="filtersDescending" [attr.alt]="intl.additionalFilters"></lu-icon>
+		</button>
+		}
+
+		<ng-content></ng-content>
+	</div>
+
+	@if (addonAfter()) {
```

Comment:
Indentation is often weird in template. Is prettier up-to-date?

Reply:
It is, that's a topic that we need to adress together at some point I think.